test_scripts/UpdateVideoStreamingCapabilities/common.lua

```diff
@@ -79,6 +81,20 @@ function m.getVideoStreamingCapability(pArraySizeAddVSC)
   return vSC
 end
 
+function m.getAnotherVideoStreamingCapability(pArraySizeAddVSC)
```

Comment:
this function is redundant and `getVideoStreamingCapability` can be used because default capabilities should be used from hmi_values.lua